### PR TITLE
Update the TM reprojecting and transforming data lesson

### PR DESCRIPTION
**Pull request by havatv**
Goal: Update the TM reprojecting and transforming data lesson (7.1)

Ticket(s): #2441

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: docs/training_manual/vector_analysis/reproject_transform.rst [hunk 1]
```diff
@@ -1,111 +1,134 @@
...
-   Click on the |setProjection| button next to the dropdown menu.
-#. The :guilabel:`CRS Selector` dialog will now appear.
+#. Click on the :guilabel:`Browse` button next to the
+   :guilabel:`File name` field
+#. Navigate to :file:`exercise_data/` and specify the name of the new
+   layer as :file:`buildings_reprojected.shp`.
+#. Change the value of the :guilabel:`CRS`.
+   Only the recent CRSs used will be shown in the drop down menu.
+   Click on the |setProjection| :sup:`Select projection` button next
+   to the dropdown menu.
```
Comment (DelazJ) [inline docs/training_manual/vector_analysis/reproject_transform.rst:127]: dropdown, drop-down, drop down
might be good we stick to a single one spelling (see 2 lines above). No?

File: docs/training_manual/vector_analysis/reproject_transform.rst [hunk 2]
```diff
@@ -1,111 +1,134 @@
...
+if you see them in the *correct* position.
 
-But you can easily export the layer in another CRS.
+You can easily export the layer with another CRS.
 
-#. Right-click on the :guilabel:`buildings` layer in the :guilabel:`Layers` panel
-#. Select :menuselection:`Export --> Save Features As...` in the menu that appears.
+#. Right-click on the ``buildings`` layer in the :guilabel:`Layers`
+   panel
+#. Select :menuselection:`Export --> Save Features As...` in the menu
```
[the remark this answers is not in the document]
Comment (havatv) [reply docs/training_manual/vector_analysis/reproject_transform.rst:117]: ? It is "Save Features As..." in QGIS 3.12 on Ubuntu 18.04.

File: docs/training_manual/vector_analysis/reproject_transform.rst (hunk 1 above)
Comment (havatv) [inline docs/training_manual/vector_analysis/reproject_transform.rst:127]:
```suggestion
   Only the recent CRSs used will be shown in the drop-down menu.
   Click on the |setProjection| :sup:`Select projection` button next
   to the drop-down menu.
```

File: docs/training_manual/vector_analysis/reproject_transform.rst [hunk 3]
```diff
@@ -123,35 +146,37 @@ are in two different CRS but they are still overlapping.
...
 
-   This projection represents the Earth on a circular field instead of a
-   rectangular one, as most other projections do.
+   This projection represents the Earth on a circular field instead of
+   a rectangular one, as most other projections do.
 
 #. Add the following string in the :guilabel:`Parameters` field::
 
-    +proj=vandg +lon_0=0 +x_0=0 +y_0=0 +R_A +a=6371000 +b=6371000 +units=m +no_defs
+     +proj=vandg +lon_0=0 +x_0=0 +y_0=0 +R_A +a=6371000 +b=6371000 +units=m +no_defs
```
[the remark this answers is not in the document]
Comment (havatv) [reply docs/training_manual/vector_analysis/reproject_transform.rst:172]: Feel free! :-)

File: docs/training_manual/vector_analysis/reproject_transform.rst [hunk 4]
```diff
@@ -1,111 +1,134 @@
...
+calculations on layer, it is always better to have the layer in a
+projected coordinate system.
 
-Be aware that the 'on the fly' reprojection is related to the **project** and not
-to single layers. This means that layers can have different CRS from the project
-even if you see them in the *correct* position.
+Be aware that the 'on the fly' reprojection is related to the
+**project** and not to single layers.
+This means that a layer can have a different CRS from the project even
+if you see them in the *correct* position.
```
Comment (havatv) [inline docs/training_manual/vector_analysis/reproject_transform.rst:111]:
```suggestion
if you see it in the *correct* position.
```

File: docs/training_manual/vector_analysis/reproject_transform.rst [hunk 5]
```diff
@@ -1,111 +1,134 @@
...
+**project** and not to single layers.
+This means that a layer can have a different CRS from the project even
+if you see it in the *correct* position.
 
-But you can easily export the layer in another CRS.
+You can easily export the layer with another CRS.
 
-#. Right-click on the :guilabel:`buildings` layer in the :guilabel:`Layers` panel
-#. Select :menuselection:`Export --> Save Features As...` in the menu that appears.
+#. Right-click on the ``buildings`` layer in the :guilabel:`Layers`
```
Comment (havatv) [inline docs/training_manual/vector_analysis/reproject_transform.rst:115]:
```suggestion
#. Add the building dataset from :file:`training_data.gpkg`
#. Right-click on the ``buildings`` layer in the :guilabel:`Layers`
```

File: docs/training_manual/vector_analysis/reproject_transform.rst [hunk 6]
```diff
@@ -1,111 +1,135 @@
...
+**project** and not to single layers.
+This means that a layer can have a different CRS from the project even
+if you see it in the *correct* position.
 
-But you can easily export the layer in another CRS.
+You can easily export the layer with another CRS.
 
-#. Right-click on the :guilabel:`buildings` layer in the :guilabel:`Layers` panel
-#. Select :menuselection:`Export --> Save Features As...` in the menu that appears.
+#. Add the building dataset from :file:`training_data.gpkg`
```
Comment (havatv) [inline docs/training_manual/vector_analysis/reproject_transform.rst:115]:
```suggestion
#. Add the ``buildings`` dataset from :file:`training_data.gpkg`
```

File: docs/training_manual/vector_analysis/reproject_transform.rst [hunk 7]
```diff
@@ -1,111 +1,135 @@
...
-that, as you travel away from the equator on your map, the scale of the objects
-that you see gets larger and larger. What this means for us, practically, is
-that there is no constant scale on our map!
-
-To solve this, let's use a Projected Coordinate System (PCS) instead. A PCS
-"projects" or converts the data in a way that makes allowance for the scale
-change and corrects it. Therefore, to keep the scale constant, we should
-reproject our data to use a PCS.
+   *Statusbar* along the bottom of the screen.
+   While over South Africa, set this value to :guilabel:`1:5 000 000`
```
Comment (havatv) [inline docs/training_manual/vector_analysis/reproject_transform.rst:25]:
```suggestion
   While over South Africa, set this value to ``1:5 000 000``
```

File: docs/training_manual/vector_analysis/reproject_transform.rst [hunk 8]
```diff
@@ -1,111 +1,134 @@
...
+**project** and not to single layers.
+This means that a layer can have a different CRS from the project even
+if you see them in the *correct* position.
 
-But you can easily export the layer in another CRS.
+You can easily export the layer with another CRS.
 
-#. Right-click on the :guilabel:`buildings` layer in the :guilabel:`Layers` panel
-#. Select :menuselection:`Export --> Save Features As...` in the menu that appears.
+#. Right-click on the ``buildings`` layer in the :guilabel:`Layers`
```
[the remark this answers is not in the document]
Comment (havatv) [reply docs/training_manual/vector_analysis/reproject_transform.rst:116]: Loading of the buildings included.

File: docs/training_manual/vector_analysis/reproject_transform.rst (hunk 1 above)
Comment (havatv) [reply docs/training_manual/vector_analysis/reproject_transform.rst]: drop-down

**Pull request merged**